it/pom.xml

```diff
@@ -100,6 +100,7 @@ for running a single test.
           <skipInvocation>${skipTests}</skipInvocation>
           <streamLogs>true</streamLogs>
           <invokerPropertiesFile>invoker.properties</invokerPropertiesFile>
+          <!--<invokerTest>*</invokerTest>-->
```

Comment:
same for me. That's why added a top-level comment in the document ;-P (not much better though)